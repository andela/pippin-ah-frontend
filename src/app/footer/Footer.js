import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { fab } from '@fortawesome/free-brands-svg-icons';
import { fas } from '@fortawesome/free-solid-svg-icons';
import { library } from '@fortawesome/fontawesome-svg-core';
import './footer.css';

library.add(fas, fab);

export default function Footer() {
  return (
    <div className="footer">
      <div className="social-media-icon-div">
        <FontAwesomeIcon
          icon={['fab', 'facebook-square']}
          style={{ fontSize: '25px', borderRadius: '50%', color: '#3B5998' }}
        />
        <FontAwesomeIcon
          icon={['fab', 'linkedin']}
          style={{ fontSize: '25px', borderRadius: '50%', color: '#65A2D9' }}
        />
        <FontAwesomeIcon
          icon={['fab', 'twitter-square']}
          style={{ fontSize: '25px', borderRadius: '50%', color: '#0E76A8' }}
        />
      </div>
      <div>
        <p style={{ fontSize: '13px' }}>copyright &copy; LearnGround 2019.</p>
      </div>
    </div>
  );
}
