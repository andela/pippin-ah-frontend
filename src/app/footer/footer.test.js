import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('HomePage Component', () => {
  it('should render the Landing Page', () => {
    const component = shallow(<Footer />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
