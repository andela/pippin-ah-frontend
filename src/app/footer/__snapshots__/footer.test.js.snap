// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render the Landing Page 1`] = `
<div
  className="footer"
>
  <div
    className="social-media-icon-div"
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "facebook-square",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      style={
        Object {
          "borderRadius": "50%",
          "color": "#3B5998",
          "fontSize": "25px",
        }
      }
      symbol={false}
      title=""
      transform={null}
    />
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "linkedin",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      style={
        Object {
          "borderRadius": "50%",
          "color": "#65A2D9",
          "fontSize": "25px",
        }
      }
      symbol={false}
      title=""
      transform={null}
    />
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "twitter-square",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      style={
        Object {
          "borderRadius": "50%",
          "color": "#0E76A8",
          "fontSize": "25px",
        }
      }
      symbol={false}
      title=""
      transform={null}
    />
  </div>
  <div>
    <p
      style={
        Object {
          "fontSize": "13px",
        }
      }
    >
      copyright Â© LearnGround 2019.
    </p>
  </div>
</div>
`;
